modules/sdk-core/src/bitgo/wallet/wallet.ts

```diff
@@ -1617,7 +1617,11 @@ export class Wallet implements IWallet {
     }
 
     if (!txPrebuild || typeof txPrebuild !== 'object') {
-      throw new Error('txPrebuild must be an object');
+      if (!txRequestId || this.multisigType() === 'onchain') {
+        throw new Error('txPrebuild must be an object');
```

Comment:
this error seems a bit misleading here no? maybe we should add more errors here and a bit more information on the error we throw, so its easier for us to debug in the future

Reply:
+1, looks like copy/paste mistake. Would give a different error message